_posts/2019-08-23-hobby-apps.md

```diff
@@ -23,13 +23,13 @@ Before answering the question of what kind of hobby app to make, the developer s
 
-2. A hobby app can provide value to myself or others that cannot be obtained from other apps. For example, as of 2013, there was no iOS app that featured the four most-important sets of United Statesian immigration laws and procedures. Immigration, an app I released that year, does this, thereby providing value to the population of users who require access to these laws and procedures, a population that has ironically not included me for some time. For another example, although there are many iOS run-tracking apps, there is no iOS run-tracking app that provides features focused on racing and training for races. RaceRunner does. I race. RaceRunner therefore provides _me_ value. There are many Spanish-verb-conjugation apps, but none taught, until the advent of Conjugar, _all_ the tenses _and_ the curious practice of [voseo](https://en.wikipedia.org/wiki/Voseo). Conjugar therefore provides value to Spanish-verb-tense completionists.
+2. A hobby app can provide value to myself or others that cannot be obtained from other apps. For example, as of 2013, there was no iOS app that featured the four most-important sets of United States immigration laws and procedures. Immigration, an app I released that year, does this, thereby providing value to the population of users who require access to these laws and procedures, a population that has ironically not included me for some time. For another example, although there are many iOS run-tracking apps, there is no iOS run-tracking app that provides features focused on racing and training for races. RaceRunner does. I race. RaceRunner therefore provides _me_ value. There are many Spanish-verb-conjugation apps, but none taught, until the advent of Conjugar, _all_ the tenses _and_ the curious practice of [voseo](https://en.wikipedia.org/wiki/Voseo). Conjugar therefore provides value to Spanish-verb-tense completionists.
```

Comment:
`United Statesian` is a [valid word](https://www.merriam-webster.com/dictionary/United%20Statesian). I don't like to use "United States" as an adjective because that word is to me a noun. `United Statesian` is better than `American`, IMO, because `American` can potentially refer to all countries in the Wester Hemisphere.